lib/mgmt_msg_native.h

```diff
@@ -237,19 +252,33 @@ _Static_assert(sizeof(struct mgmt_msg_tree_data) ==
 #define GET_DATA_FLAG_STATE	0x01	/* get only "config false" data */
 #define GET_DATA_FLAG_CONFIG	0x02	/* get only "config true" data */
 #define GET_DATA_FLAG_EXACT	0x04	/* get exact data node instead of the full tree */
+/*
```

Comment:
Also while in here, let's remove `only` from the comments above since that's not what is actually happening. These _are_ flags and you can have both set in which case both "config" and "data" are returned not "only data" and "only config" :)

Reply:
Done.